src/actions/userstories.js

```diff
@@ -51,7 +51,7 @@ export const deleteUserstory = (dispatch, projectId, userStoryId) => {
   axios
     .delete(`/apiprojects/projects/${projectId}/userstories/${userStoryId}`)
     .then((res) => {
-      if (res.status) {
+      if (res.status == 200) {
```

Comment:
@TitusCln 
I'm not seeing the backend specifying responses, this is done by the framework so validating if the response was 200 it's letting room for issues...
As I see we have two options:
1. Remove this validations, because if we get an error it will go to catch
2. Specify in the backend this expected response.

Reply:
This one is still being validated